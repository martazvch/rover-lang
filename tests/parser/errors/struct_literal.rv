Foo{45}
// For synchronization
var a = 1
Foo{a

var a = Foo{a:5}
var a = Foo{
