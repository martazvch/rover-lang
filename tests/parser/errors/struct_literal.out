struct_lit_non_ident_field
expect_equal_struct_lit
expect_equal_struct_lit
expect_brace_after_struct_lit
